tests/release/pipelines/e2e-test-default-bundle.go

```diff
@@ -1,24 +1,23 @@
-package release
+package pipelines
 
 import (
+	"encoding/json"
 	"fmt"
+	tektonutils "github.com/davidmogar/release-service/tekton/utils"
+	"k8s.io/apimachinery/pkg/runtime"
 
-	"github.com/devfile/library/v2/pkg/util"
+	releaseApi "github.com/davidmogar/release-service/api/v1alpha1"
 	ecp "github.com/enterprise-contract/enterprise-contract-controller/api/v1alpha1"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	appservice "github.com/redhat-appstudio/application-api/api/v1alpha1"
-	"github.com/redhat-appstudio/e2e-tests/pkg/constants"
 	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
 	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
-	"github.com/redhat-appstudio/e2e-tests/pkg/utils/release"
 	"github.com/redhat-appstudio/e2e-tests/pkg/utils/tekton"
-	releaseApi "github.com/redhat-appstudio/release-service/api/v1alpha1"
-	"gopkg.in/yaml.v2"
 	corev1 "k8s.io/api/core/v1"
 )
 
-var _ = framework.ReleaseSuiteDescribe("[HACBS-738]test-release-service-default-pipeline", Label("release", "defaultBundle", "HACBS"), func() {
+var _ = framework.ReleaseSuiteDescribe("[HACBS-738]test-release-service-default-pipeline", Label("release-pipelines", "defaultBundle", "HACBS"), func() {
```

Comment:
It would be nice to rename this. We can do it in a followup PR (and address the duplicated constants there too) but since bundles are gone, having bundle in the name is not great and I don't love calling it "default." But yeah, feel free to resolve and we can address in a followup that isn't tied to the API changes